tests/integration/security/sds_ingress/util/util.go

```diff
@@ -644,3 +644,34 @@ func SetInstances(apps echo.Services) error {
 	}
 	return nil
 }
+
+// Some cloud platform may throw the following error during creation of the service with mixed TCP/UDP protocols:
+// "Error syncing load balancer: failed to ensure load balancer: mixed protocol is not supported for LoadBalancer".
+// Make sure the service is up and running before proceeding with the test.
+func WaitForIngressQUICService(t framework.TestContext, ns string) error {
```

Comment:
Is the goal to test just the service corresponding to the ingress gateway to be running/ready? Right now we're checking all services in the specified namespace are running. Or is it easier to generalize?

Reply:
Hello @jaellio 

The goal is to make sure that the platform, that the cluster is running on top, supports a mixed (QUIC) service.
Unfortunately, when I made the tests on aws cloud platform, which is not support the mixed protocol services, the service just stuck in a "pending" state forever.

It has not other status / indicator that I could track to know if the mixed protocol on service is supported.
The only way, is to check the events of the service, which has a pretty straight forward error message that could indicate about the issue.